src/engine/ScreenOverlays.tscn

```diff
@@ -13,7 +13,7 @@ pause_mode = 2
 process_priority = 2000
 
 [node name="CanvasLayer" type="CanvasLayer" parent="."]
-layer = 128
+layer = 127
```

Comment:
Yeah it's pretty tricky since the render priority of these two scenes sometimes intertwine. Perhaps the color filter could be in its own dedicated canvas layer which renders above all.

Reply:
What's the overlay that *shouldn't* be on top of everything else?

Reply:
There's render hierarchy even in ScreenOverlays.

Reply:
But that's just an effect of the scene order they are defined, right? So it is not relevant when discussing that all-non overlays should be underneath the overlays. Except I guess exclusive popups really should be on top.

Maybe reserving layer 126 for normal overlays would be good, then exclusive popups can be 127, and finally absolutely on top of everything being effect layers would be 128?

Reply:
Sounds good.